components/server/src/auth/rate-limiter.ts

```diff
@@ -220,6 +220,8 @@ const defaultFunctions: FunctionsConfig = {
     getIDEOptions: { group: "default", points: 1 },
     getPrebuildEvents: { group: "default", points: 1 },
     setUsageAttribution: { group: "default", points: 1 },
+    getUsageLimitFor: { group: "default", points: 1 },
```

Comment:
nit: I would have expected `getUsageLimit` instead of `getUsageLimitFor`

Reply:
Renamed